CONTRIBUTING.rst

```diff
@@ -162,3 +162,138 @@ their individual contributions.
 * `Will Hall <https://www.github.com/wrhall>`_ (`wrsh07@gmail.com <mailto:wrsh07@gmail.com>`_)
 * `Will Thompson <https://www.github.com/wjt>`_ (`will@willthompson.co.uk <mailto:will@willthompson.co.uk>`_)
 * `Zac Hatfield-Dodds <https://www.github.com/Zac-HD>`_ (`zac.hatfield.dodds@gmail.com <mailto:zac.hatfield.dodds@gmail.com>`_)
+
+
+-----------------
+Review Guidelines
+-----------------
+
+All changes to Hypothesis go through a code review process, and must be
+signed off by at least one person with write access to the repo other
+than the author of the change. This document
+outlines some guidelines on what to look for and what we're interested
+in. It applies uniformly to maintainers and external contributors.
+
+At a high level, the two things we're looking for in review are answers
+to the following questions:
+
+1. Is this change going to make users' lives worse?
+2. Is this change going to make the maintainers' lives worse?
+
+Code review is a collaborative process between the author and the
+reviewer to try to ensure that the answer to both of those questions
+is no.
+
+Ideally of course the change should also make one or both of the users'
+and our lives *better*, but it's OK for changes to be mostly neutral.
+The author should be presumed to have a good reason for submitting the
+change in the first place, so neutral is good enough!
+
+The rest of this section outlines specific questions reviewers should
+ask in aid of this, but the above is the most important part and where
+the guidelines below clash with the objectives above, the above wins
+out.
+
+When submitting changes it can be helpful to go through a pre-review
+pass of this yourself just to see what people will look at, but it's
+not required.
+
+~~~~~~~~~~~~~~~~~
+General Questions
+~~~~~~~~~~~~~~~~~
+
+The following are useful questions to ask for almost every change:
+
+1. Is this change too large? Could it profitably be broken up into
+   several smaller changes that could be judged on their individual
+   merits?
+2. Is the motivation for this change clear?
```

Comment:
Clear to who?  Suggest "Has the motivation for this change been clearly explained?", with a target audience of non-DRMacIver maintainers (generally familiar with Hypothesis, but not assuming deep knowledge of internals).

Then again, I'm basically saying "explain this to me please" so it's not an unbiased requirement 😉